cli/pack/pack_test.go

```diff
@@ -3,6 +3,7 @@ package pack
 import (
 	"fmt"
 	"io/ioutil"
+	"log"
```

Comment:
Why did you start using `log`, AFAIU all the messages in the test we output with `assert`?

Reply:
You are right. Removed.